### ISSUE TITLE
Make date of onset mpx data from ECDC available in machine readable format

### ISSUE THREAD

**Issue by Mougk** [globaldothealth/monkeypox#103]
ECDC published a report showing countries case time series based on symptom onset dates. These are relevant data for real-time analyses of growth rates.

Proposal: make data available in machine readable format via global.health github.

Link to report here: https://monkeypoxreport.ecdc.europa.eu/

<img width="998" alt="Screenshot 2022-06-24 at 11 11 48" src="https://user-images.githubusercontent.com/8478551/175517092-65607775-01be-4941-ac5b-31cbab913f90.png">

Some helpful pointers here: https://twitter.com/uffe1974/status/1540275194903887873?s=20&t=0uZZZolGanfRFLUh7BnQMA

**Comment by badrange** [comment]
FYI: ECDC also makes data available as xls, csv and json here, apparently updated twice a week:
https://www.ecdc.europa.eu/en/publications-data/data-monkeypox-cases-eueea

**Comment by badrange** [comment]
From the csv file:
```
DateRep,CountryExp,CountryCode,Source,ConfCases
2022-04-26,Austria,AT,EI,0
2022-04-26,Belgium,BE,TESSy,0
2022-04-26,Czechia,CZ,TESSy,0
2022-04-26,Denmark,DK,EI,0
2022-04-26,Finland,FI,EI,0
2022-04-26,France,FR,EI,0
2022-04-26,Germany,DE,EI,0
2022-04-26,Greece,EL,TESSy,0
2022-04-26,Hungary,HU,TESSy,0
2022-04-26,Iceland,IS,EI,0
2022-04-26,Ireland,IE,TESSy,0
2022-04-26,Italy,IT,TESSy,0
```

**Comment by Mougk** [comment]
Agree, would be good to also have these to estimate delays in reporting from onset --> confirmation.
